### DISCUSSION TITLE
Dictionary for languages

### DISCUSSION THREAD

**Discussion by zetrozky** [monkeytypegame/monkeytype#908] [Ideas] [answered]
With monkeytype getting English 450k, it's probably a good idea to let people know exactly what they're typing. Users may be able to hover over words on the summary screen after a test and know what they typed.

Edit: Also sources for quotes

Edit 2: Sources for quotes have apparently been added already

**Comment by ghost** [comment, marked as answer]
Both are available, by now. I am somewhat convinced they were added before you asked, but there is a sources list "Yuki, Midorikawa" and the text at the bottom, which shows if you click toggle words history  button. If you hover over it it shows you what you typed.

There is also a button for copying them to clipboard, as the app prevents you from selecting the words.

![GO](https://user-images.githubusercontent.com/55037535/106562025-d1328b00-657d-11eb-8175-add133898975.png)
Edit: The quote checks out. That was not intended lol.

**Reply by zetrozky** [reply]
The sources are nice but I don't think there's a dictionary, word history and copying to clipboard is currently the only way to know the definition of the words you type.

**Reply by frogfrogfrogfrogfrog** [reply]
@ZETROzky I don't see the point of having a dictionary for words. Isn't the point to just be able to type those hard words faster? Even so having a dictionary definition for each and every single word doesn't really seem feasible. As always you can just copy and paste the words you don't know but there's not really a point in knowing what your typing if the point is to just type harder words faster.

**Reply by zefr0x** [reply]
Maby just redirect the user to another website to get the meaning from a dictionary like [wordreference](https://www.wordreference.com).

We can request the meaning in the URL `https://www.wordreference.com`/`sourceLang&&anotherLang`/`word`
Example: `https://www.wordreference.com/enar/Example`
